schemas/1.6.0/adaptive-card.json

```diff
@@ -2168,6 +2222,34 @@
         }
       ]
     },
+    "InputLabelPosition": {
+      "description": "Specifies position of the label with respect to input.",
+      "anyOf": [
+        {
+          "enum": [
+            "inline",
+            "above"
+          ]
+        },
+        {
+          "pattern": "^([i|I][n|N][l|L][i|I][n|N][e|E])|([a|A][b|B][o|O][v|V][e|E])$"
+        }
+      ]
+    },
+    "InputStyle": {
+      "description": "`revealOnHover`: Show input fields in read only view unless user takes any action. `default`: Show the default editable state for input controls",
```

Comment:
we need to let mobile users know that this feature is not available to their users.

Reply:
Indicated!